scrub/header/header.go

```diff
@@ -0,0 +1,54 @@
+package header
```

Comment:
Put this in `scrub/header.go`, no need to make a `header` sub package. This would import as header and you would call `header.Header`, which makes no sense really where as `scrub.Header` does make sense.